backend/addcorpus/models.py

```diff
@@ -264,14 +266,15 @@ def clean(self):
 
     @property
     def has_named_entities(self):
-        client = elasticsearch(self.es_index)
+        from es.search import total_hits
```

Comment:
Is there a reason this line isn't at the top of the file?

Reply:
Yes, otherwise it's a circular import, as the `es.search` module imports `addcorpus.models.CorpusConfiguration`.